### DISCUSSION TITLE
brave: How to block ~/Downloads?

### DISCUSSION THREAD

**Discussion by firejailaddssecuirty** [netblue30/firejail#5073] [Q&A] [answered]
I use the Brave browser only for Youtube and Facebook. I never download any files using Brave. I want to remove the ~/Downloads folder access. How do I do that ?

**Comment by ghost** [comment, marked as answer]
> I want to remove the ~/Downloads folder access.

If you follow the chain of included files starting in /etc/firejail/brave.profile you'll see that it [redirects](https://github.com/netblue30/firejail/blob/master/etc/profile-a-l/brave.profile#L37) to chromium-common.profile. It is in that file that ${DOWNLOADS} is [whitelisted](https://github.com/netblue30/firejail/blob/master/etc/profile-a-l/chromium-common.profile#L29).

To override an option Firejail expects a counter-option to arrive BEFORE that specific option (see [this](https://github.com/netblue30/firejail/wiki/Creating-overrides) for a general introduction). So, with this info you can put a `nowhitelist ${DOWNLOADS}` option into either `~/.config/firejail/brave.local` or `~/.config/firejail/chromium-common.local`. The choice is yours, but if you happen to use another chromium-based web browser it's better to go for brave.local. Otherwise the ~/Downloads folder will be inaccessible in all apps that include chromium-common.profile.

> I use the Brave browser only for Youtube and Facebook.

You might want to consider using Firejail's `private` option if you want to discard all modifications when closing Brave. I'd say experiment with those options combined and check if it suits your use case.

**Reply by firejailaddssecuirty** [reply]
It worked.
I added `nowhitelist ${DOWNLOADS}`  to `~/.config/firejail/brave.local`
